content/en/docs/how-tos/rbac.md

```diff
@@ -44,6 +44,57 @@ No API is provided to create Rover groups. We have to do some manual work on the
 The existing groups defined in the release repo will be removed from the release repo and the clusters. 
 If such a group is used as a subject in a RBAC manifest, then its owner has to ensure the group also exists on Rover. In case that the group name cannot be managed by the owner, e.g., the desired group name is taken by other teams, then the RBAC manifest has to use another Rover group.
 
+For example, `group/cvp-pool-admins` cannot exist any more in the release repo because it contains usernames.
+
+```yaml
+apiVersion: v1
+kind: List
+items:
+- apiVersion: user.openshift.io/v1
+  kind: Group
+  metadata:
+    name: cvp-pool-admins
+  users:
+  - bob
+  - jim
+- apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Don't show this binding in the example, you are not talking about it

Reply:
It is talked about now. :)